LEGO1/mxtransitionmanager.h

```diff
@@ -2,19 +2,58 @@
 #define MXTRANSITIONMANAGER_H
 
 #include "mxcore.h"
+#include "mxvideopresenter.h"
```

Comment:
Based on a cursory glance, I think some of those includes can be moved to the `mxtransitionmanager.cpp` unit to cut down on dependencies.